### ISSUE TITLE
After a restart, containers created pre-upgrade cannot be pinged by other containers

### ISSUE THREAD

**Issue by soumyalj** [rancher/rancher#6731]
**Rancher Version:**1.1.4 to 1.2.0-pre4-rc12

**Docker Version:**1.1.4

**OS and where are the hosts located? (cloud, bare metal, etc):**Ubuntu, Cloud

**Setup Details: (single node rancher vs. HA rancher, internal DB vs. external DB)**

**Environment Type: (Cattle/Kubernetes/Swarm/Mesos)**Cattle

**Steps to Reproduce:**
1. Create a cattle setup in v1.1.4, create services/lbs
2. Upgrade rancher to v1.2.0-pre4-rc12
3. Upgrade the env

**Results:** 
 After upgrade, restart a container created before upgrade. The container restarted is not reachable by other containers.
```
root@c0050ddc3e55:/# ping Default_service1_1
PING default_service1_1.rancher.internal (10.42.112.194): 48 data bytes
^C--- default_service1_1.rancher.internal ping statistics ---
89 packets transmitted, 0 packets received, 100% packet loss

```

**Comment by deniseschannon** [comment]
Fixed in plugin-manager:v0.2.8

**Comment by soumyalj** [comment]
Tested with upgrade from 1.1.4 to 1.2.0-pre4-rc14. The issue described above is not seen. Containers created pre-upgrade can be pinged by other containers after a restart.
